webserver/main.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	"github.com/gin-gonic/gin"
+	"gopkg.in/yaml.v2"
+)
+
+type Config struct {
+	Server struct {
+		Port string `yaml:"port"`
+		Host string `yaml:"host"`
+	} `yaml:"server"`
+	Database struct {
+		Username string `yaml:"user"`
+		Password string `yaml:"pass"`
+	} `yaml:"database"`
+}
+
+type Wedstrijd struct {
+	Datum         string   `json:"Datum"`
+	Plaats        string   `json:"Plaats"`
+	LSR           bool     `json:"LSR"`
+	MSR           bool     `json:"MSR"`
+	KSR           bool     `json:"KSR"`
+	Jeugd         bool     `json:"Jeugd"`
+	BSR           bool     `json:"BSR"`
+	Kwalificatie  bool     `json:"Kwalificatie"`
+	Afstanden     []string `json:"Afstanden"`
+	MinLeeftijd   string   `json:"min.leeftijd"`
+	Organisator   []string `json:"Organisator"`
+	Inschrijflink []string `json:"Inschrijflink"`
+	Uitslag       []string `json:"Uitslag"`
+	ONK           bool     `json:"ONK"`
+	Koppel        bool     `json:"Koppel"`
+	Afgelast      bool     `json:"Afgelast"`
+	Coords        []string `json:"coords"`
+}
+
+var wedstrijden []Wedstrijd
+
+func main() {
+	//The config file is gitignored, copy from config_defaults.yaml and edit
+	config, err := decodeConfig("../config.yaml")
+	if err != nil {
+		fmt.Print(err)
+		return
+	}
+
+	//For now, import wedstrijden once from a json file
```

Comment:
This is to bootstrap the database right or how will this work with updates over time? (probably out of scope for now)

Reply:
This will be replaced by setting up the database connection (without pre-loading anything). That connection is then used in the API handling functions to retrieve or update the information